src/main/java/racingcar/service/SystemService.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.service;
+
+import racingcar.vo.Car;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class SystemService extends Util {
+
+    public List<Car> carInfo = new ArrayList<>();
+
+    private static final int RANDOM_NUMBER = 10;
+    private static final int MOVABLE_NUMBER = 4;
```

Comment:
```suggestion
    private static final int RANDOM_NUMBER = 10;
    private static final int MOVABLE_NUMBER = 4;
    
    public List<Car> carInfo = new ArrayList<>();
```
1) 상수를 클래스의 최 상단에 위치시키고, 2) 클래스 변수, 3)필드 변수(= 멤버 변수) 순으로 위치하게 됩니다.  위와 같이 순서에 맞게 변경해주세요 🙏 

carInfo 필드 변수는 접근제어자를 ``public``으로 하신 이유가 있을까요? 🤔 

[이전 코멘트](https://github.com/next-step/java-racingcar/pull/2175#discussion_r623826688)도 참고해서 반영해주세요 🙏 
[[객체지향 설계] (캡슐화\)](https://velog.io/@y_dragonrise/객체지향-설계-캡슐화Encapsulation) 해당 자료를 참고해보면 좋을 것 같아요 😉